docs/source/grpo_trainer.md

```diff
@@ -24,6 +24,7 @@ This example demonstrates how to train a model using the GRPO method. We train a
 ></iframe>
 
 Below is the script to train the model.
+Note that the actual train batch size is `num_generations * per_device_train_batch_size * gradient_accumulation_steps`. Additionally, the input tensor for the forward pass has a size of `num_generations * per_device_train_batch_size`. Since GRPO generates `num_generations` completions for each batch prompt, adjusting these values appropriately can help prevent OOM errors.
```

Comment:
this is standard terminology i think:
```suggestion
Note that the effective train batch size is `num_generations * per_device_train_batch_size * gradient_accumulation_steps`. Additionally, the input tensor for the forward pass has a size of `num_generations * per_device_train_batch_size`. Since GRPO generates `num_generations` completions for each batch prompt, adjusting these values appropriately can help prevent OOM errors.
```

Reply:
```suggestion
Note that the input tensor for the forward pass has a size of `num_generations * per_device_train_batch_size` because GRPO generates `num_generations` completions for each prompt in the batch. Adjusting these values appropriately can help prevent OOM errors.
Consequently, the effective train batch size is `num_generations * per_device_train_batch_size * gradient_accumulation_steps`.
```